README.md

```diff
@@ -1,41 +1,277 @@
+| `/accounts/finalize-registration`  | `accounts-finalize-registration`  | No                          | Finalize registration page for accounts                           |
+| `/accounts/create-account`         | `accounts-create-account`         | No                          | Create account page for accounts                                  |
+| `/accounts/create-password`        | `accounts-create-password`        | No                          | Create password page for accounts                                 |
+| `/accounts/forgot-password`        | `accounts-forgot-password`        | No                          | Forgot password page for accounts                                 |
+| `/accounts/verify-email`           | `accounts-verify-email`           | No                          | Email verification page for accounts                              |
+| `/accounts/forgot-password/reset`  | `accounts-forgot-password-reset`  | No                          | Reset password page for accounts                                  |
+| `/accounts/invitation/accept`      | `accounts-accept-invitation`      | No                          | Accept invitation page for accounts                               |
+| `/accounts`                        | `accounts`                        | Yes                         | Main account page with child routes                               |
+| `/accounts/:urlFriendlyName`       | `accounts-Default`                | Yes                         | Default account page (redirect to dashboard)                      |
+| `/accounts/:urlFriendlyName/users` | `accounts-users`                  | Yes                         | User management page for accounts                                 |
+| `/accounts/:urlFriendlyName/me`    | `accounts-me`                     | Yes                         | Profile page for the account owner                                |
+| `/accounts/:urlFriendlyName/change-password` | `accounts-changePassword` | Yes                         | Change password page for the account owner                        |
+| `/accounts/:urlFriendlyName/add-password`  | `accounts-add-password` | Yes                         | Add password page for the account owner                           |
+| `/accounts/:urlFriendlyName/change-email`  | `accounts-changeEmail`  | Yes                         | Change email page for the account owner                           |
+| `/accounts/:urlFriendlyName/settings`      | `accounts-settings`      | Yes                         | Settings page for the account owner                               |
+| `/accounts/:urlFriendlyName/account`       | `accounts-account`       | Yes                         | Account details page for the account owner                        |
+| `/redirect-to-login-message`      | `redirectToLoginMessage`         | No                          | Redirect to login message page                                    |
```

Comment:
This table is great, because it reveals naming inconsistencies with the route names. Some examples:
 - accounts-accept-invitation
 - accounts-Default
 - accounts-changePassword
 - redirectToLoginMessage

These are 4 different styles. As I see it, the most widely used one is the kebab-case with lower-case letters, just like the first example. Let's use that convention.